pkgs/applications/terminal-emulators/blackbox-terminal/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, fetchpatch
+, meson
+, ninja
+, desktop-file-utils
+, gettext
+, glib
+, gtk4
+, gtk3
+, libadwaita
+, libmarble
+, marble
+, pkg-config
+, vala
+, vte
+, json-glib
+, pcre2
+, libxml2
+, librsvg
+, cmake
+, wrapGAppsHook4
+, python3
+, qt5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "blackbox-terminal";
+  version = "0.12.0";
+
+  strictDeps = false;
```

Comment:
This is the default behaviour.

Reply:
Yes, but the build breaks if it is enabled, which might be a future default.

Reply:
Then there should probably be a comment explaining the build failure.